src/zjs_console.c

```diff
@@ -152,6 +159,82 @@ static jerry_value_t console_error(const jerry_value_t function_obj,
     return do_print(function_obj, this, argv, argc, stderr);
 }
 
+static jerry_value_t console_time(const jerry_value_t function_obj,
+                                  const jerry_value_t this,
+                                  const jerry_value_t argv[],
+                                  const jerry_length_t argc)
+{
+    if (!jerry_value_is_string(argv[0])) {
+        ERR_PRINT("invalid parameters\n");
+        return ZJS_UNDEFINED;
+    }
+    int sz = jerry_get_string_size(argv[0]);
+    char label[sz + 1];
```

Comment:
You've got to watch the limit the maximum size of the string here, as usual, or we can overflow the stack.